package.json

```diff
@@ -69,7 +69,7 @@
     "worker-loader": "^0.8.0"
   },
   "engines": {
-    "node": ">=4.3.0 <5.0.0 || >=5.10"
+    "node": ">=6.11.5"
```

Comment:
Is webpack 4 going to remove drop support of node v4?

Reply:
Yes

Reply:
Why? Seems too soon.

Release | LTS Status | Codename | Active LTS Start | Maintenance LTS Start | Maintenance LTS End
-- | -- | -- | -- | -- | --
4.x | Maintenance LTS | Argon | 2015-10-01 | 2017-04-01 | April 2018